packages/react-server-test-pages/pages/navigation/data-bundle-cache.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import DataBundleCache from "react-server-data-bundle-cache";
+import {
+	Link,
+	RootContainer,
+} from "react-server";
+import {
+	GET,
+	ROWS,
+	PagePointer,
+	RowIndex,
+	RowMS,
+	ClientRenderIndicator,
+} from "./common";
+
+require("./common.css");
+require("./data-bundle-cache.css");
+
+DataBundleCache.install();
```

Comment:
Just to make sure I'm understanding: `DataBundleCache.install()` is something we'd typically put somewhere central, rather than per-page, right?